### PR TITLE
Added description for XDebug configuration

### PR DESCRIPTION
**Pull request by JohnKlein1991**
Issue: https://github.com/phpdocker-io/phpdocker.io/issues/192

### CODE REVIEW

File: src/PHPDocker/Template/README.md.twig [hunk 1]
```diff
@@ -71,3 +71,38 @@ It's hard to avoid file permission issues when fiddling about with containers du
...
+# Simple basic Xdebug configuration with integration to PHPStorm
+
+## To config xdebug you need add these lines in php-fpm/php-ini-overrides.ini:
+
+### For linux:
+xdebug.remote_enable=1
+
+xdebug.remote_connect_back=1
+
+xdebug.remote_autostart = 1
```
Comment (luispabon) [inline src/PHPDocker/Template/README.md.twig:84]:
```suggestion
```
xdebug.remote_enable=1
xdebug.remote_connect_back=1
xdebug.remote_autostart = 1
```
```

File: src/PHPDocker/Template/README.md.twig [hunk 2]
```diff
@@ -71,3 +71,38 @@ It's hard to avoid file permission issues when fiddling about with containers du
...
+
+### For MacOS and Windows:
+xdebug.remote_enable=1
+
+xdebug.remote_host=host.docker.internal
+
+xdebug.remote_autostart = 1
```
Comment (luispabon) [inline src/PHPDocker/Template/README.md.twig:91]: Please enclose in three back ticks to enable code highlighting.

**Review by luispabon — CHANGES_REQUESTED** [review]
Just a couple of styling changes

**Comment by JohnKlein1991** [comment]
> Just a couple of styling changes

Did it

**Comment by luispabon** [comment]
Thank you

**Pull request merged**